packages/projection-typeorm/src/operators/storeStakeKeys.ts

```diff
@@ -7,6 +7,6 @@ export const storeStakeKeys = typeormOperator<Mappers.WithStakeKeys>(
     const repository = queryRunner.manager.getRepository(StakeKeyEntity);
     const createAll = insert.length > 0 ? repository.insert(insert.map((hash) => ({ hash }))) : Promise.resolve();
     const deleteAll = del.length > 0 ? repository.delete(del) : Promise.resolve();
-    return Promise.all([createAll, deleteAll]);
```

Comment:
Here, more than changing `return` into `await` I would remove `async` from the function definition: the signature doesn't change and there is one less Promise to handle (minor minor minor optimization)